### DISCUSSION TITLE
Is this Project Dead ?

### DISCUSSION THREAD

**Discussion by theCJMan** [tgalal/yowsup#3197] [Q&A]
Hi there
Is this Project Dead, I see it was last updated in 2021 / 12 / 14 -> More than 12 months ago

And if it is, what is the next project to use for WhatsApp Business API integration?

**Comment by theCJMan** [comment]
@ben221199 Do you know?

**Reply by ben221199** [reply]
- I don't know if Yowsup is dead. I think @tgalal can tell you more about it.
- Also, Chat-API of @mgp25 is gone, but [libsignal library](https://github.com/mgp25/libsignal-php) is still there.
- My project, WAPI, is paused, because I simply don't have time for it. Also, WAPI only implements the basics, like Noise protocol and FunXMPP encoding/decoding. Because I don't want users to abuse the software for spamming, developers should implement E2E themselves.
